tools/xcodeproj_shims/install.sh

```diff
@@ -10,9 +10,9 @@ find "${BAZEL_DIAGNOSTICS_DIR}" -type f -atime +7d -delete
 
 case "${PRODUCT_TYPE}" in
 com.apple.product-type.framework)
-    input_options=("bazel-bin/$BAZEL_BIN_SUBDIR/${TARGET_NAME}/${FULL_PRODUCT_NAME}")
+    input_options=("bazel-bin$BAZEL_BIN_SUBDIR/${TARGET_NAME}.framework")
```

Comment:
Do you know why this changed?

```suggestion
    input_options=("bazel-bin/$BAZEL_BIN_SUBDIR/${TARGET_NAME}.framework")
```